coalib/bearlib/languages/definitions/TinyBasic.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class TinyBasic:
+    aliases = 'tb',
+    extensions = '.tb',
+    comment_delimiters = 'REM', "'"
+    string_delimiters = {'"': '"', "'": "'"}
+    encapsulators = {'(': ')'}
+    keywords = [
```

Comment:
I think there are other keywords also like NAP, CLS etc. You can refer here http://tinybc.sourceforge.net